templates/resource-tracker/tracker-manager.hbs

```diff
@@ -0,0 +1,54 @@
+<form class='{{cssClass}}' tracker-manager autocomplete='off'>
+  <div class='resource-editor' style='position: relative'>
+    <div style='display: flex; flex-flow: column nowrap;'>
+      <div style='overflow-y: auto; height: 250px;'>
+        <div class='tracker-manager table'>
+          {{#if templates}}
+            <div class='label'>{{localize 'GURPS.resourceTemplateName'}}</div>
+            <div class='label'>{{localize 'GURPS.resourceTemplateAlias'}}</div>
+            <div class='label'>{{localize 'GURPS.autoApply'}}</div>
+            <div class='label'>{{localize 'GURPS.resourceTemplateInitializer'}}</div>
+            <div><a id='template-add' class='button icon'><i class='fas fa-plus'></i></a></div>
+          {{else}}
+            <div
+              style='grid-column: 1 / 6; justify-content: center; font: italic bold larger Roboto; color: darkslateblue;'
+            >
+              {{localize 'GURPS.resourceTemplateClickToAdd'}}
+              <a id='template-add' class='button icon'><i class='fas fa-plus'></i></a>
+            </div>
+          {{/if}}
+
+          {{#each templates}}
+            <!-- tracker.name -->
+            <input
+              name='name'
+              data='{{@index}}'
+              class='gcs-input clickable'
+              type='text'
+              readonly
+              value='{{tracker.name}}'
+              placeholder='{{localize "GURPS.clickToEdit"}}'
+            />
+            <!-- tracker.alias -->
+            <input name='alias' data='{{@index}}' class='gcs-input' type='text' readonly value='{{tracker.alias}}' />
+
+            <!-- slot -->
+            <input name='autoapply' name='slot' type='checkbox' data='{{@index}}' {{checked slot}} />
```

Comment:
the autoapply value is failing to save. You have two name properties defined here, I think one of them is a mistake.